dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -527,6 +535,11 @@ class ECSCluster(SpecCluster):
         and this operation takes a while.
 
         Default ``False``.
+    platform_version: str (optional)
+        Version of the AWS Fargate platform to use, e.g. "1.4.0" or "LATEST". This
```

Comment:
If None, this gets the value from the config - what happens if it is not set in the config, do you get LATEST?

Reply:
It's ok if it's not set in the config and stays as None -- that would cause the code to skip L211 and just not set `platformVersion` in kwargs. That will cause AWS itself to will use LATEST. (AWS does that if you don't set a platformVersion. e.g. They have LATEST set to "1.3.0" right now, even though "1.4.0" is available.)